src/wolfkrow/core/tasks/nuke_render.py

```diff
@@ -15,7 +15,7 @@
 from wolfkrow.core.engine.resolver import Resolver
 
 class NukeTask(Task):
-    def export_to_command_line(self, job_name, temp_dir=None, deadline=False):
+    def export_to_command_line(self, job_name, temp_dir=None, deadline=False, export_json=False):
```

Comment:
This was also supposed to go out on the latest Develop branch... I must have screwed up git stuff somewhere...